ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/ListUserRightsWithGivenUserIdAuthIT.scala

```diff
@@ -30,8 +30,6 @@ class ListUserRightsWithGivenUserIdAuthIT
       // create a normal user
       (alice, _) <- createUserByAdmin(testId + "-alice", identityProviderId)
 
-      // test that only admins can retrieve his own user and the newly created alice user
-      _ <- getRights("participant_admin")
```

Comment:
Do we cover this case through `AdminServiceCallAuthTests`?

Reply:
let me create a test case for that

Reply:
I dont see how its different from this case here: https://github.com/digital-asset/daml/blob/main/ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/ListAuthenticatedUserRightsAuthIT.scala#L50

Reply:
:+1: